utils/acpica/Makefile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=acpica
+PKG_VERSION:=20170531
+PKG_RELEASE:=1
+
+PKG_BASENAME:=$(PKG_NAME)-unix
```

Comment:
I would suggest just using the expanded form `acpica-unix`.  IMHO, names starting with `PKG_` should be avoided for local use.